### ISSUE TITLE
How to evaluate the model ?

### ISSUE THREAD

**Issue by haibin894609937** [anantzoid/VQA-Keras-Visual-Question-Answering#4]
I train the model ,and How to evaluate it

**Comment by saifhassan** [comment]
To evaluate the model on validation set, run python train.py --type val.
